daemon/graphdriver/vfs/driver.go

```diff
@@ -32,18 +35,22 @@ func (d *Driver) String() string {
 	return "vfs"
 }
 
+// Status of the driver should be returned, a nil array is returned for VFS driver.
```

Comment:
Status is used for implementing the `graphdriver.ProtoDriver` interface. `VFS` does not currently have any status information.